plugins/gameobjects/blitter/blitterbase/BlitterBase.d.ts

```diff
@@ -64,11 +67,11 @@ declare class BlitterBase extends Phaser.GameObjects.GameObject {
     blendMode: Phaser.BlendModes | string;
     setBlendMode(value: string | Phaser.BlendModes): this;
 
+    width: number;
+    height: number;
     displayWidth: number;
     displayHeight: number;
-    setSize(width: number, height: number): this;
     setDisplaySize(width: number, height: number): this;
-    resize(width: number, height: number): this;
```

Comment:
As I understand the logic of this file, first we describe custom props & methods, then we described inherited props & methods.

So, `resize` from `ComputedSize` is replaced with custom `resize` here:

https://github.com/rexrainbow/phaser3-rex-notes/blob/master/plugins/gameobjects/blitter/blitterbase/BlitterBase.js#L109
https://github.com/rexrainbow/phaser3-rex-notes/blob/master/plugins/gameobjects/blitter/blitterbase/methods/Methods.js#L12
https://github.com/rexrainbow/phaser3-rex-notes/blob/master/plugins/gameobjects/blitter/blitterbase/methods/Resize.js

At the same time, `setSize` is just an alias for this custom `resize`. So, I've moved both of them to the top of this file.